### PR TITLE
add whitespace analyzer docs

### PR DESCRIPTION
**Pull request by AntonEliatra**
### Description
add whitespace analyzer docs

### Issues Resolved
Part of https://github.com/opensearch-project/documentation-website/issues/1483 addressed in this PR.

### Version
all

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by vagimeli** [comment]
@udabhas @varun-lodaya Please provide tech review approval to move this PR forward in the documentation process. Please review this week or provide a peer who can review it. Thank you.

**Review by udabhas — APPROVED** [review]
Looks good to me!

File: _analyzers/whitespace.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+---
+layout: default
+title: Whitespace analyzer
+nav_order: 60
+---
+
+# Whitespace analyzer
+
+The `whitespace` analyzer breaks text into tokens based only on whitespace characters (spaces, tabs, and others). It does not apply any transformations, such as lowercasing or removing stopwords, therefore the case of the original text is retained and punctuation is included as part of the tokens.
```
Comment (natebower) [inline _analyzers/whitespace.md:9]:
```suggestion
The `whitespace` analyzer breaks text into tokens based only on white space characters (for example, spaces and tabs). It does not apply any transformations, such as lowercasing or removing stopwords, so the original case of the text is retained and punctuation is included as part of the tokens.
```

**Review by natebower — COMMENTED** [review]
@kolchfa-aws @AntonEliatra Editorial review. Thanks!

**Review by kolchfa-aws — APPROVED** [review]

**Pull request merged**